src/Scene/Scene.js

```diff
@@ -215,18 +190,14 @@ Scene.prototype.scene3D = function () {
  *
  * @param node {[object Object]}
  */
-Scene.prototype.add = function (node, nodeProcess) {
-    if (node instanceof Globe) {
-        this.map = node;
-        nodeProcess = nodeProcess || new NodeProcess(this.currentCamera(), node.ellipsoid);
-        // this.quadTreeRequest(node.tiles, nodeProcess);
-    }
+Scene.prototype.add = function (layer, updater) {
+    this.layers.push({ layer, updater });
+    this.gfxEngine.add3DScene(layer.getMesh());
+};
 
-    this.layers.push({
-        node,
-        process: nodeProcess,
-    });
-    this.gfxEngine.add3DScene(node.getMesh());
+Scene.prototype.setMap = function (layer, updater) {
```

Comment:
Add a comment answering these questions: why/when should I use `setMap` ? why/when should I use `add` ?

Also is this `this.map` thing necessary at all?